tests/ParserTest.elm

```diff
@@ -558,53 +573,132 @@ expr =
+                  , singleQuote "😃"
                   , Ok (Literal (Char '😃'))
                   )
                 , ( "escaped unicode code point"
-                  , "'\\u{1F648}'"
+                  , singleQuote "\\u{1F648}"
                   , Ok (Literal (Char '🙈'))
                   )
                 ]
               )
             , ( "literal string"
               , [ ( "empty"
-                  , "\"\""
+                  , doubleQuote ""
                   , Ok (Literal (String ""))
                   )
                 , ( "one space"
-                  , "\" \""
+                  , doubleQuote " "
                   , Ok (Literal (String " "))
                   )
                 , ( "two numbers"
-                  , "\"42\""
+                  , doubleQuote "42"
                   , Ok (Literal (String "42"))
                   )
                 , ( "single quote"
-                  , "\"'\""
+                  , doubleQuote "'"
                   , Ok (Literal (String "'"))
                   )
+                , ( "double quote"
+                  , doubleQuote "\\\""
+                  , Ok (Literal (String "\""))
+                  )
+                , ( "escape n"
+                  , doubleQuote "\\n"
+                  , Ok (Literal (String "\n"))
+                  )
+                , ( "escape r"
+                  , doubleQuote "\\r"
+                  , Ok (Literal (String "\u{000D}"))
+                  )
+                , ( "escape t"
+                  , doubleQuote "\\t"
+                  , Ok (Literal (String "\t"))
+                  )
+                , ( "emoji"
+                  , doubleQuote "😃"
+                  , Ok (Literal (String "😃"))
+                  )
+                , ( "escaped unicode code point"
+                  , doubleQuote "\\u{1F648}"
+                  , Ok (Literal (String "🙈"))
+                  )
+                , ( "combo of escapes and chars"
+                  , doubleQuote "\\u{1F648}\\n\\r\\t\\abc123"
+                  , Ok (Literal (String "🙈\n\r\tabc123"))
+                  )
+                ]
+              )
+            , ( "literal multiline string"
+              , [ ( "empty"
+                  , tripleQuote ""
+                  , Ok (Literal (String ""))
+                  )
+                , ( "one space"
+                  , tripleQuote " "
+                  , Ok (Literal (String " "))
+                  )
+                , ( "newline"
+                  , tripleQuote "\n"
+                  , Ok (Literal (String "\n"))
+                  )
+                , ( "two numbers"
+                  , tripleQuote "42"
+                  , Ok (Literal (String "42"))
+                  )
+                , ( "single quote"
+                  , tripleQuote "'"
+                  , Ok (Literal (String "'"))
+                  )
+                , ( "double quote"
+                  , tripleQuote " \" "
+                  , Ok (Literal (String "\""))
```

Comment:
I'd expect the spaces to be in the parsed string too

Reply:
Fixed by https://github.com/elm-in-elm/compiler/pull/31/commits/51d0b7fbad7f45642e392142f5acad0f1d6ad1cc